packages/oauth/src/openshift/osoauth-service.ts

```diff
@@ -137,7 +145,7 @@ export class OSOAuthService implements OAuthProtoService {
     const beforeSend = (xhr: JQueryXHR, settings: JQueryAjaxSettings) => {
       if (tokenHasExpired(this.userProfile)) {
         log.debug(`Cannot navigate to ${settings.url} as token expired so need to logout`)
-        doLogout(config)
+        this.doLogout(config)
```

Comment:
Is this a fix for #189?

Reply:
No it is not. It is necessary to bring `doLogout` in as a member method so that the `unregister` function provided by the fetch-interceptor is available to it. #189 will be investigated separately.